Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:3.8.5-slim
+FROM python:3.9.10
```

It should be fine to relax the python version unless you have a reason not to.

```suggestion
FROM python:3.9
```